notebooks/setup_code.ipynb

```diff
@@ -19,6 +19,27 @@
     "from aiidalab_widgets_base import AiiDACodeSetup, CodeDatabaseWidget"
    ]
   },
+  {
```

Comment:
Is there an actual change in this notebook or is this just random Jupyter shuffling?